core/services/mongodb-remote/src/main/java/com/mongodb/stitch/core/services/mongodb/remote/sync/internal/DataSynchronizer.java

```diff
@@ -138,7 +138,7 @@ public DataSynchronizer(
     this.listenersLock = new ReentrantLock();
     this.eventDispatcher = eventDispatcher;
     this.instanceKey = instanceKey;
-
+    this.dispatchGroup = new DispatchGroup();
```

Comment:
[naming] I think this should be a bit more descriptive, maybe `ongoingOperationsGroup`?